### DISCUSSION TITLE
Cursor question

### DISCUSSION THREAD

**Discussion by Whoola117** [praydog/REFramework#508] [General Questions]
In Monster Hunter Ries is there any way to remove the windows cursor from the screen? Or alternatively is there a way to put REFramework "under" The game instead on top? 

I hope what I'm asking is understood, if not let me know and I will see if I can ask in a different way.

**Comment by praydog** [comment]
Just close the REFramework menu and it should go away.

**Reply by Whoola117** [reply]
Cool, I wasn't sure if I kept working after I hit the X and closed it, thanks for the information.
